Hotline/macOS/TrackerView.swift

```diff
@@ -459,6 +459,16 @@ struct TrackerItemView: View {
   }
 }
 
+#if DEBUG
+private struct TrackerViewPreview: View {
```

Comment:
Once you're on Xcode 16 min, this can be replaced with @Previewable: https://developer.apple.com/documentation/SwiftUI/Previewable()